<launch>

    <include file="$(find offboard_control)/launch/sensors/rs_t265.launch"/>

    <node name="position_to_mavros" pkg="offboard_control" type="position_to_mavros_node" respawn="true" output="screen">
        <rosparam file="$(find offboard_control)/config/position_to_mavros_params.yaml" command="load"/>
    </node>

    <node name="safe_offboard" pkg="offboard_control" type="safe_offboard_node" respawn="true" output="screen">
        <rosparam file="$(find offboard_control)/config/safe_offboard_params.yaml" command="load"/>
    </node>

    <include file="$(find mavros)/launch/px4.launch">
        <arg name="fcu_url" default="/dev/pixhawk:57600" />
    </include>

    <!--<include file="$(find offboard_control)/launch/pc_filter.launch" />-->

</launch>
